packages/run-v5/lib/colorize.js

```diff
@@ -1,4 +1,4 @@
-const {default: ux} = require('cli-ux')
+require('@oclif/core')
```

Comment:
How do you feel about explicitly importing CliUx?
```suggestion
const {CliUx} = require('@oclif/core')
```